resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
+    <DirectoryRef Id="<%= component[:directory_ref] %>">
+      <Component Id='<%= component[:id] %>' Guid="<%= component[:guid] %>" <%= "Win64=\"$(var.Win64)\"" unless component[:win64].nil? %>>
+        <%- if component[:file][:id] == "NSSM" -%>
```

Comment:
We shouldn't be putting in service specific code in here.
So if a <CreateFolder> is needed (bearing in mind James' comment below), we should do that with a specific tag like :create_folder => "true".